tests/test_matlabify.py

```diff
@@ -138,12 +138,19 @@ def test_classes(mod):
     assert isinstance(cls, doc.MatClass)
     assert cls.getter("__name__") == "ClassInheritHandle"
     assert cls.getter("__module__") == "test_data"
-    assert cls.bases == ["handle", "my.super.Class"]
+    assert cls.bases == [("handle",), ("my", "super", "Class")]

```

Comment:
Could this be changed back to the original. A list of fully-qualified names?

Reply:
Yea this can absolutely be changed back. I don't _think_ this broke anything and I was thinking it may be useful for later processing but now with fresh eyes I am unconvinced.